File: semestr-5/ASD 1/PRO/programy-z-cwiczen/2019/C02/TZ03-IS1-ConsoleApplication_0002/TZ03-IS1-ConsoleApplication_0002/MyUtility.cs
//213.135.45.43:7070
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyUtilityNamespace
{
    class MyUtility
    {
        public static void MyPause(string myStr = null)
        {
            Console.Write(myStr);
            Console.ReadKey();
        }
        public static int MyRead(string myStr="x ?= ")
        {
            Console.Write(myStr);
            return Convert.ToInt32(Console.ReadLine());
        }
        public static Random myFate = new Random();
    }
}

File: semestr-5/Algorytmy i struktury danych 1/PRO/programy-z-cwiczen/2018/C06/01_ID03P05/PRO_C06_prg_0001/MyUtility.cpp
int MyRead(){
    return MyRead("x? = ");
    }
///********************************************************
int MyRead(string myStr){
    int x;
    cout<<myStr;
    cin>>x;
    return x;
    }
///********************************************************
int MyRead(string myStr, istream& s){
    int x;
    cout<<myStr;
    s>>x;
    return x;
    }

